priv/repo/migrations/20170424091310_soft_delete_user.exs

```diff
@@ -0,0 +1,9 @@
+defmodule CoursePlanner.Repo.Migrations.SoftDeleteUser do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :deleted_at, :utc_datetime
```

Comment:
I was talking to @ghatighorias about this type. What do you think of using `naive_datetime`?
The standard timestamps (inserted_at, updated_at) use this type